packages/dice-webapp/components/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,163 @@
+"use client";
+import React, { useState } from "react";
+import Image from "next/image";
+import LogoPath from "@/public/DICE.svg";
+import { IoMenuOutline } from "react-icons/io5";
+import { FaDiceD20 } from "react-icons/fa";
+import { usePathname } from "next/navigation";
+import { cn } from "@nextui-org/system";
+import { signOut } from "next-auth/react";
+import { Button } from "@nextui-org/button";
+import { IoMdLogOut } from "react-icons/io";
+import { orgSidebarData } from "./sidebar-data";
+import Link from "next/link";
+
+function Hover(
+  props: React.HtmlHTMLAttributes<HTMLDivElement> & { isActive?: boolean },
+) {
+  return (
+    <div
+      {...props}
+      className={cn(
+        "py-2 px-3 hover:bg-gray-200 dark:hover:bg-zinc-900  transition-all cursor-pointer rounded-sm",
+        {
+          "dark:border-b-1 dark:border-b-white border-b-1 border-b-black rounded-sm":
+            props.isActive,
+        },
+        props.className,
+      )}
+    />
+  );
+}
+
+type Props = {};
+
+export default function Sidebar({}: Props) {
+  const [isSidebarOpen, setSidebarOpen] = useState(false);
+  function toggleSidebar() {
+    setSidebarOpen(!isSidebarOpen);
+  }
+  return (
+    <div
+      className={cn(
```

Comment:
Just specify the className=`the string of styles` directly, instead of wrapping it in `cn()`

Reply:
Required changes were made.